src/core/dbus-manager.c

```diff
@@ -48,6 +50,10 @@
 #include "virt.h"
 #include "watchdog.h"
 
+/* Require 16MiB free in /run/systemd for reloading/reexecing. After all we need to serialize our state there, and if
+ * we can't we'll fail badly. */
+#define RELOAD_DISK_SPACE_MIN (UINT64_C(16) * UINT64_C(1024) * UINT64_C(1024))
```

Comment:
Not that it matters in any way, but for the learning exercise: do the casts make any practical difference?